src/coreHandler.ts

```diff
@@ -75,9 +75,11 @@ export class CoreHandler {
 		})
 		this.core.onDisconnected(() => {
 			this.logger.info('Core Disconnected!')
+			this.setStatus(P.StatusCode.WARNING_MAJOR, ['Core Disconnected'])
```

Comment:
```suggestion
```
As @ianshade mentions, I don't think that it makes sense to try to tell core that we are disconnected from it.